src/services/org-admin.js

```diff
@@ -580,4 +583,69 @@ module.exports = class OrgAdminService {
 			console.log(error)
 		}
 	}
+
+	/**
+	 * Bulk create users
+	 * @method
+	 * @name bulkUserCreate
+	 * @param {Array} users - user details.
+	 * @param {Object} tokenInformation - token details.
+	 * @returns {CSV} - created users.
+	 */
+
+	static async bulkSessionCreate(filePath, tokenInformation) {
+		try {
+			const { id, organization_id } = tokenInformation
+			const userDetail = await userRequests.details('', id)
```

Comment:
@sumanvpacewisdom Add this call after validating the csv input.

Reply:
@rakeshSgr this is the details of the user who is uploading the csv not details of csv and also the csv validation is done after sending it to queue.